### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by ntadimeti**
update links to the pretrained models and link to the NGC catalog

### CODE REVIEW

File: tutorials/pretrained_models.rst [hunk 1]
```diff
@@ -4,9 +4,8 @@ Pretrained_models
 This is a list of AtacWorks denoising models that were reported in Lal & Chiang, 2019. 
 Models were trained to denoise bulk or single-cell ATAC-seq data of varying sequencing depths.
 
-Note that all the AtacWorks models described in here are available to `download <https://ngc.nvidia.com/catalog/models>`_.
-
-To download a model and it's corresponding data, search for the name at "Model path" column in the NGC model catalog. You can find a wget command at the bottom of the description box of the model as shown in the screenshot below.
+To download a model and it's data, Click the link in "Model path" column in the table :ref:`Pretrained Models Table`.
```
Comment (avantikalal) [inline tutorials/pretrained_models.rst:7]: it's -> its

File: tutorials/pretrained_models.rst [hunk 2]
```diff
@@ -4,9 +4,8 @@ Pretrained_models
...
+The link will take you to the NGC catalog where you can find a wget command at the bottom of the description box of the model as shown in the screenshot below.
 
 .. image:: ngc_catalog_screenshot.png
    :width: 800
```
Comment (avantikalal) [inline tutorials/pretrained_models.rst:11]: Line 14: subfloder -> subfolder

**Review by avantikalal — CHANGES_REQUESTED** [review]

File: tutorials/pretrained_models.rst (hunk 1 above)
Comment (ntadimeti) [reply tutorials/pretrained_models.rst]: done

File: tutorials/pretrained_models.rst (hunk 2 above)
Comment (ntadimeti) [reply tutorials/pretrained_models.rst:11]: done

File: tutorials/pretrained_models.rst [hunk 3]
```diff
@@ -24,29 +23,119 @@ Briefly, each ATAC-seq read was converted to a single genomic position correspon
 
 Before using these models, please note that they should only be used on data that is processed exactly as described above. If your bigWig files and/or peak calls are generated using a different method, these models may give unreliable results, and it would be preferable to train your own model as described in Tutorial 1.
```
Comment (avantikalal) [inline tutorials/pretrained_models.rst:24]: Can we link to tutorial 1 here?

**Review by avantikalal — CHANGES_REQUESTED** [review]

File: tutorials/pretrained_models.rst (hunk 3 above)
Comment (ntadimeti) [reply tutorials/pretrained_models.rst]: done.

**Review by avantikalal — APPROVED** [review]

**Pull request merged**